src/qp/metrics/pit.py

```diff
@@ -163,7 +163,7 @@ def evaluate_PIT_outlier_rate(self, pit_min=0.0001, pit_max=0.9999):
         float
             The percentage of outliers in this distribution given the min and max bounds.
         """
-        return calculate_outlier_rate(self._pit, pit_min, pit_max)
+        return calculate_outlier_rate(self._pit, pit_min, pit_max)[0]
```

Comment:
This was a small bug that I noticed, and is unrelated to the vectorization of the metrics. Prior to this change, the results of this function look like this: `[x]`. Now it's this: `x`, which is in alignment with what was previously expected.